sympy/polys/rings.py

```diff
@@ -2308,7 +2363,47 @@ def imul_num(self, c):
             self[exp] *= c
         return self
 
-    def _rem(self, G):
+    def _rem(self, g):
```

Comment:
This is duplicating everything from `_rem_list`. Is there some advantage here in terms of efficiency compared to just doing this the obvious way:
```
def _rem(self, g):
     return self._rem_list([g])
```